### PR TITLE
update news

### PR DESCRIPTION
**Pull request by ycohn**

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Releasing paperclip
+Releasing paperclip
```
Comment (tute) [inline RELEASING.md]: There is new trailing white space here. Can you please remove it?

File: NEWS [hunk 2]
```diff
@@ -1,5 +1,8 @@
-master:
-
+5.0.0.beta1:
```
Comment (tute) [inline NEWS]: Nitpick: Let's add new trailing whitespace between the title and the links?

File: NEWS [hunk 3]
```diff
@@ -1,5 +1,8 @@
...
+* Removed support for AWS v1 
+* Rails Versioning: Removed support for Rails 3.2 and 4.1
```
Comment (tute) [inline NEWS]: > * Rails Versioning: Removed support for Rails 3.2 and 4.1

I'd phrase this as "Remove support for end-of-life'd Rails 3.2 and 4.1".

File: NEWS [hunk 4]
```diff
@@ -1,5 +1,8 @@
...
+* Extracted repository for locales only:  https://github.com/thoughtbot/paperclip-i18n
+* Removed tests for JRuby and Rubinius in Travis CI
```
Comment (tute) [inline NEWS]: "Removed tests for JRuby and Rubinius _from_ Travis CI" (from instead of in?)

**Comment by tute** [comment]
There are at least these more changes to add to the NEWS:

```
* `url` on a unpersisted record returns `default_url` rather than `nil`
* Bugfix for image magick scaling images up
```

**Comment by tute** [comment]
Some changes to make: I'd make it the present tense for consistency with the rest of the file.

I'd move together all the removals:

```
* Drop support for end-of-life'd Ruby 2
* Remove tests for JRuby and Rubinius from Travis CI (they were failing)
* Remove support for end-of-life'd Rails 3.2 and 4.1
* Remove support for AWS v1 
```

And, more items to add:

```
* Bugfix: Original file could be unlinked during `post_process_style`, producing failures
* Memory consumption improvements
```

Thanks so much! This is great work.

File: NEWS [hunk 5]
```diff
@@ -1,6 +1,14 @@
...
+
+* Memory consumption improvements
+* Extracted repository for locales only:  https://github.com/thoughtbot/paperclip-i18n
+* `url` on a unpersisted record returns `default_url` rather than `nil`
+* Bugfix: Original file could be unlinked during `post_process_style`, producing failures
+* Bugfix for image magick scaling images up
+* Drop support for end-of-life'd Ruby 2
+* Remove tests for JRuby and Rubinius from Travis CI (they were failing)
+* Remove support for end-of-life'd Rails 3.2 and 4.1
+* Remove support for AWS v1
```
Comment (tute) [inline NEWS]: Nitpick: I think "Drop" will read better than "Remove".

**Comment by tute** [comment]
Can you please squash your commits together? Let me know if you need help.
Then I'll merge.
Thanks for your help!

**Comment by tute** [comment]
Squashed for you, releasing 5.beta1 soon. Thanks for your help!

**Comment by ycohn** [comment]
Thanks!

**Pull request closed without merging**